README.md

```diff
@@ -1,6 +1,6 @@
 # Math Magicians
 > This project is the Microverse's math_magicians project. 
-> Its a web application for use react that is a imple calculator. 
+> Its a web application for use react that is a simple calculator. 
 
 
```

Comment:
### In the `src/components/calculator.js`
- [ **OPTIONAL** ] **RECOMMENDED:** For JavaScript best practices. I think it would be better if you use an **Arrow function** instead of s normal function. i.e your function should look like this:

```JavaScript
const Calculator = () => (
    // reference as a js object
      <div>
       .... your code here....
      </div>
);
```

- [ **OPTIONAL** ] **RECOMMENDED:** According to **React's**  convention, every comment is supposed to start with a capital letter. So, I recommend you edit your `src/components/calculator.js` and `src/components/Calculator.js` components files to be `src/components/Calculator.js` and `src/components/Buttons.js` respectively in order to follow the **React's**  convention. For more information about this, you can read this **[article](https://iq.js.org/questions/react/why-should-component-names-start-with-capital-letter)**